units/systemd-pcrextend@.service.in

```diff
@@ -0,0 +1,19 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=TPM2 PCR Extension (Varlink)
+Documentation=man:systemd-pcrphase.service(8)
+DefaultDependencies=no
+Conflicts=shutdown.target initrd-switch-root.target
+Before=shutdown.target initrd-switch-root.target
+
```

Comment:
The .socket unit has that already, and this template unit only is useful in that context. If the socket doesn't start the service doesn't matter...

conditions on socket-activated services are a bit weird: we should already condition ot the sockets always, so that service activation then always works. putting conditions on the services instead just creates bugs I am sure, because things fail weirdly